rootfs/usr/local/bin/eks-update-kubeconfig

```diff
@@ -10,6 +10,7 @@
 # We expect AWS_DEFAULT_SHORT_REGION was set in Dockerfile, and if not there,
 # then in aws.sh, but just in case...
 AWS_DEFAULT_SHORT_REGION=${AWS_DEFAULT_SHORT_REGION:-$(aws-region --${AWS_REGION_ABBREVIATION_TYPE} ${AWS_DEFAULT_REGION:-us-west-2})}
+TENANT_LABEL_ENABLED=false
```

Comment:
This raises a larger issue of how we support tenants in Geodesic (on the command line/console) versus how we support them via Spacelift. Do we add an environment variable in Dockerfile like we have for `NAMESPACE` and do we add tools for switching tenants along the lines of `assume-role`? We need to have a design sprint for this. @osterman @korenyoni @aknysh @nitrocode